Userland/Services/DHCPClient/DHCPv4.h

```diff
@@ -31,7 +31,7 @@ enum class DHCPOption : u8 {
     // BOOTP
     Pad = 0,
     SubnetMask,
-    TimeOffset,
+    Duration,
```

Comment:
looks more than suspicious.

Reply:
Yep, though luckily nothing uses this option at the moment.